lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
The current version overwrote the [default trackers of createTorrent](https://github.com/feross/create-torrent/blob/master/index.js#L4), so only the ones which support the websocket protocol kept enabled. Using a chromium based implementation instead of `wrtc` it should expect the same support as the regular browser version.